src/Plugin/formatter/FormatterJson.php

```diff
@@ -27,17 +39,17 @@ public function prepare(array $data) {
 
     $output = array('data' => $data);
```

Comment:
I agree 100% on this. This would be a path to remove the coupling between the two objects.

Care to create an issue and link here?